great_expectations/validation_operators/validation_operators.py

```diff
@@ -378,11 +380,17 @@ def _run_actions(
                     validation_result_suite_identifier=validation_result_id,
                     validation_result_suite=batch_validation_result,
                     data_asset=batch,
+                    payload=batch_actions_results,
                 )
 
+                # add action_result
                 batch_actions_results[action["name"]] = (
                     {} if action_result is None else action_result
                 )
+                batch_actions_results[action["name"]]["class"] = action["action"][
+                    "class_name"
+                ]
+
```

Comment:
We should make it clear somewhere in docs that the order of actions is important, as the payload will only have the output payloads of previous actions.